cloudigrade/account/migrations/0001_initial.py

```diff
@@ -54,7 +54,7 @@ class Migration(migrations.Migration):
             name='AwsAccount',
             fields=[
                 ('account_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to='account.Account')),
-                ('aws_account_id', models.DecimalField(db_index=True, decimal_places=0, max_digits=12)),
+                ('aws_account_id', models.CharField(db_index=True, max_length=12)),
```

Comment:
Speaking as a newcomer to Django, I think because Django is confusing in parts to newcomers. @blentz Think of these migrations files as timestamped history. When we modify anything on a model you can run `python path/to/cloudigrade/manage.py makemigrations` and it will autogen a new file for you with your update. You can manually then run the `migrate` command or use `make reinitdb` and your DB will update.